src/server/database/utilities.js

```diff
@@ -2,6 +2,16 @@
 const r = require('rethinkdb');
 const config = require('./config');
 
+let connectionPromise = null;
+
+function connect() {
```

Comment:
@maxkfranz Do you think it makes sense to use lodash.memoize here?

Reply:
This is about caching the connection to rethinkdb.

Reply:
Yes, that would work

Reply:
@gaelder learn about memoize https://lodash.com/docs/4.17.4#memoize

And use that to cache the rethinkdb connection.

Reply:
@d2fong Done. Let me know if I did something weird.